recipes/fastgltf/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import get
+from conan.tools.build import check_min_cppstd
+
+class fastgltf(ConanFile):
+    name = "fastgltf"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/spnda/fastgltf"
+    description = "A blazing fast C++17 glTF 2.0 library powered by SIMD."
+    topics = ("gltf", "simd", "cpp17")
+
+    # Needed for CMake configurations for dependencies to work
+    generators = "CMakeDeps"
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    exports_sources = "CMakeLists.txt", "src/*", "cmake/*"
+
+    options = {
+        "fPIC": [True, False],
+        "useSmallVector": [True, False],
```

Comment:
As per the recommended naming (not required but it's easier for consumer when they are consistent -- requires less thinking and less coffee)

```suggestion
        "enable_small_vector": [True, False],
```